pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -188,6 +200,10 @@ func StatefulSetForCluster(
 										cluster.Namespace,
 									),
 								},
+								{
+									Name:  "CASSANDRA_ENDPOINT_SNITCH",
+									Value: "GossipingPropertyFileSnitch",
```

Comment:
Perhaps make this a const so it's easy to see which snitch we use?

Reply:
:+1: